encoders/numeric/UMAPEncoder/manifest.yml

```
@@ -0,0 +1,14 @@
+manifest_version: 1
+name: UMAPEncoder
+kind: encoder
+description:
+  |
+  Encodes data using Uniform Manifold Approximation and Projection Embedding.
+author: Will Perkins (hi@willperkins.com)
+url: https://jina.ai
+vendor: Jina AI Limited
+documentation: https://github.com/jina-ai/jina-hub
```

The link will later be used at hub.jina.ai and therefore we need to have it pointing to the proper page.
```suggestion
documentation: https://github.com/jina-ai/jina-hub/blob/master/encoders/numeric/UMAPEncoder/README.md
```